source/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1 class="heading-large">GOV.UK Design System</h1>
+<main id="content" role="main">
+  <%= partial 'components/masthead' %>
+  <div class="govuk-mb-5 govuk-u-core-19">
+    <div class="govuk-o-site-width-container app-c-site-width-container">
+        <%= partial 'components/three-blocks' %>
+        <div class="govuk-o-grid">
+          <div class="govuk-o-grid__item govuk-o-grid__item--full">
+            <%= partial 'components/divider' %>
```

Comment:
Would placing this outside of the grid simplify the code? It feels a little strange to start the grid and then immediately used a full width element, but understand if it's needed to make the padding / margins work correctly.

Reply:
I agree and have moved the divider out of the grid accordingly, it's not needed for margins. Thanks.